pylint_odoo/checkers/no_modules.py

```diff
@@ -358,6 +364,8 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    _name_computed_methods = []
```

Comment:
If you use this variable when is affected and is not cleaned... so the variable is changed.

I mean,
<img width="346" alt="screen shot 2017-09-08 at 18 28 51" src="https://user-images.githubusercontent.com/6644187/30234283-489dfbb6-94c1-11e7-8bf6-4969167bf64e.png">


Better you should use a object variable like as `self.variable`

Reply:
@moylop260 
Thanks for teach me this behaviour, I fixed my little mistake